src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDeployment.java

```diff
@@ -52,15 +52,25 @@ public void deploy(DeployConfiguration config) throws AppEngineException {
     Preconditions.checkNotNull(config);
     Preconditions.checkNotNull(config.getDeployables());
     Preconditions.checkArgument(config.getDeployables().size() > 0);
+    File workingDirectory = null;
```

Comment:
I can do it here or in the else branch of the next if statement. Just saving a line, I guess. :)